<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timesheet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/logo.svg</normaloff>../Icons/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#DCE5FE&quot;;</string>
   </property>
   <widget class="QLabel" name="lblDateFrom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFrom">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblDateTo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMachCode">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mach Code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMach">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="TimeSheetTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1441</width>
     <height>605</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Poppins SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Header Design */

QHeaderView::section {
	font-family: Poppins;
	font-weight: bold;
	font-size: 13px;
	color: white;
	background-color: &quot;#344273&quot;;
	border: none;
}

/* Style for the selected row */
QTableView::item:selected:active {
	 selection-background-color: &quot;#9DADDF&quot;;
}

/* Style for the alternate row */
QTableWidget {
	alternate-background-color: &quot;#DDDDDD&quot;;
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>17</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>31</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Bio No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Emp Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Emp Name</string>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cost Center</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Days Work</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Days Present</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hours Work</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ordinary Day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ordinary Day OT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ordinary Day ND</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ordinary Day ND OT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Regular Holiday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Regular Holiday ND</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Regular Holiday ND OT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Late</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Absent</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Posted</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="sideBar">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>0</y>
     <width>261</width>
     <height>665</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#sideBar{
	background-color: qlineargradient(spread:pad, x1:0.508, y1:0.0340909, x2:0.503051, y2:1, stop:0 rgba(33, 44, 89, 255), stop:0.497175 rgba(52, 66, 115, 255), stop:0.847458 rgba(72, 87, 142, 255))
}

#btnInsertDeduction{
background-color: &quot;#546aba&quot;;
color: white;
border: none;
border-radius: 5px;
}

#btnInsertDeduction:hover{
background-color: #485ba1;
}

#btnPayTrans{
border-radius: 5px;
color: white;
background-color: &quot;#30d977&quot;;
}

#btnPayTrans:hover{
	background-color: &quot;#26ad5f&quot;;
}

#btnSendToEmail{
border-radius: 5px;
color: white;
background-color: rgb(255, 110, 112);
}

#btnSendToEmail:hover{
background-color: #c75657;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="searchBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchBar{
	border: 1px solid &quot;#344273&quot;;
	border-radius: 3px; 
}
*{
background-color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txtSearch_4">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtSearch{

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Type to search...</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchIcon_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchIcon{
	background-color: white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icons/search.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
